common/culture/traditions/00_regional_traditions.txt

```diff
@@ -834,21 +860,16 @@ tradition_desert_nomads = {
 	}
 	
 	is_shown = {
-		OR = {
-			like_culture_bedouin_trigger = yes
-			any_parent_culture = {
-				has_cultural_tradition = tradition_desert_nomads
-			}
-		}
+		# Warcraft
+		any_culture_county = {
+			any_county_province = { geographical_region = custom_deserts }
+		}	
 	}
+	
 	can_pick = {
-		custom_tooltip = {
-			text = culture_in_camel_region_desc
-			any_culture_county = {
-				any_county_province = {
-					geographical_region = world_innovation_camels
```

Comment:
It's important to check camel region because this tradition gives bonuses to camels.

Reply:
The regions for `world_innovation_camels` are included in `custom_deserts`. I can leave it in but it would be checking twice right?

Reply:
`custom_deserts` includes Voldun that doesn't have camels.

Reply:
How about `world_kalimdor_south_silithus` and `world_innovation_camels`?

Reply:
> How about `world_kalimdor_south_silithus` and `world_innovation_camels`?

The problem is there's no camels outside `world_innovation_camels` so the camel bonuses there are useless.

Reply:
That's true but the tradition includes other bonuses too:
```
	parameters = {
		desert_warrior_trait_is_better = yes
	}
	character_modifier = {
		camel_cavalry_maintenance_mult = -0.2
		camel_cavalry_recruitment_cost_mult = -0.2
	}
	province_modifier = {
		desert_tax_mult = 0.1
	}
```

Reply:
Unfortunately this is the only [vanilla tradition](https://ck3.paradoxwikis.com/Traditions) that buffs both desert provinces and camels. `tradition_saharan_nomads` only applies to Oasis provinces. How about moving the `desert_tax_mult` from `tradition_desert_nomads` to `tradition_saharan_nomads`? This effectively turns one into a strictly economic tradition and the other into a strictly combat tradition.

Reply:
Ok, but prob need to tweak localization as it mentions camels.